lib/fileutils.rb

```diff
@@ -1764,12 +1756,44 @@ def fu_get_gid(group)   #:nodoc:
   end
   private_module_function :fu_get_gid
 
+  # Updates modification times (mtime) and access times (atime)
+  # of the entries given by the paths in +list+;
+  # by defaault, creates an empty file for any path to a non-existent entry:
   #
-  # Updates modification time (mtime) and access time (atime) of file(s) in
-  # +list+.  Files are created if they don't exist.
+  #   # Single string path.
+  #   f = File.new('src0.txt') # Existing file.
+  #   f.atime # => 2022-06-10 11:11:21.200277 -0700
+  #   f.mtime # => 2022-06-10 11:11:21.200277 -0700
+  #   FileUtils.touch('src0.txt')
+  #   f = File.new('src0.txt')
+  #   f.atime # => 2022-06-11 08:28:09.8185343 -0700
+  #   f.mtime # => 2022-06-11 08:28:09.8185343 -0700
   #
-  #   FileUtils.touch 'timestamp'
-  #   FileUtils.touch Dir.glob('*.c');  system 'make'
+  #   # Array of string paths.
+  #   FileUtils.touch(['src0.txt', 'src0.dat'])
+  #
+  #   # Pathname.
+  #   require 'pathname'
+  #   path = Pathname.new('src0.txt')
+  #   FileUtils.touch(path)
+  #
+  # Keyword arguments:
+  #
+  # - <tt>mtime: <i>time</i></tt> - sets the entry's mtime to the given time,
+  #   instead of the current time.
+  # - <tt>nocreate: true</tt> - raises an exception if the entry does not exist.
+  # - <tt>noop: true</tt> - does not entries; returns +nil+.
```

Comment:
This sounds a bit odd.

```suggestion
  # - <tt>noop: true</tt> - does not touch entries; returns +nil+.
```